.github/workflows/ci.yml

```
@@ -30,13 +31,21 @@ jobs:
             target: x86_64-pc-windows-msvc
             features: '--no-default-features --features rodio_backend,pancurses_backend,share_clipboard,notify'
     steps:
+      - uses: actions/checkout@v3
+      - uses: actions/cache@v3
+        name: Cache build data
+        with:
+          path: |
+            ~/.cargo/registry
+            ~/.cargo/git
+            ~/.cargo/bin
+            target
```

```suggestion
            ~/.cargo/bin/
            ~/.cargo/registry/index/
            ~/.cargo/registry/cache/
            ~/.cargo/git/db/
            target/
```
Ah, thanks for taking another look at this. This is the path list currently listed in the `actions/cache` examples. Maybe it's more specific?

---

I don't know what is in the registry, or where I found that path exactly. It worked fine so I think it's ok, but maybe being more specific could help prevent problems by keeping files that could cause problems with toolchain/package updates out of the cache. Either one could be tested for a bit and if it doesn't work, the other one can be used 🙂

---

I'll just commit them, I'd assume since they're listed in the `actions/cache` docs they should be fine